client/src/tests/__snapshots__/MovieContainer.test.js.snap

```diff
@@ -52,214 +52,7 @@ exports[`renders correctly 1`] = `
-              </div>
-              <div
-                className="inline"
-              >
-                Kill Bill: Vol1
-              </div>
-              <div
-                className="inline margin-left"
-              >
-                October 10, 2003
-              </div>
-              <div>
-                Actions&Advanture
-              </div>
-            </div>
-            <div
-              className="movie"
-              onClick={[Function]}
-            >
-              <div
-                className="image"
-              >
-                <img
-                  src="test-file-stub"
-                />
-              </div>
-              <div
-                className="inline"
-              >
-                Jackie Brown
-              </div>
-              <div
-                className="inline margin-left"
-              >
-                December 25, 1997
-              </div>
-              <div>
-                Drama
-              </div>
-            </div>
-            <div
-              className="movie"
-              onClick={[Function]}
-            >
-              <div
-                className="image"
-              >
-                <img
-                  src="test-file-stub"
-                />
-              </div>
-              <div
-                className="inline"
-              >
-                Four rooms
-              </div>
-              <div
-                className="inline margin-left"
-              >
-                December 9, 1995
-              </div>
-              <div>
-                Comedy
-              </div>
-            </div>
-            <div
-              className="movie"
-              onClick={[Function]}
-            >
-              <div
-                className="image"
-              >
-                <img
-                  src="test-file-stub"
-                />
-              </div>
-              <div
-                className="inline"
-              >
-                Pulp fiction
-              </div>
-              <div
-                className="inline margin-left"
-              >
-                September 10, 1994
-              </div>
-              <div>
-                Oscar-winning movies
-              </div>
-            </div>
-            <div
-              className="movie"
-              onClick={[Function]}
-            >
-              <div
-                className="image"
-              >
-                <img
-                  src="test-file-stub"
-                />
-              </div>
-              <div
-                className="inline"
-              >
-                Reservoir dogs
-              </div>
-              <div
-                className="inline margin-left"
-              >
-                January 21, 1992
-              </div>
-              <div>
-                Independent movies
-              </div>
-            </div>
-          </div>
+          No films found
```

Comment:
_trivial_
Have there any changes happened in the business logic of your App, so you have different snapshots? If the newly created snapshot is in align with the current behaviour of your App, then OK, you can ignore this.